### PR TITLE
`getErrorLogs` endpoint returns errors from the `RepaymentRouter`

### PR DESCRIPTION
**Pull request by saturnial**

### CODE REVIEW

File: src/types/error_parser.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import * as _ from "lodash";
+
+import { Logging } from "./logging";
+import { DebtKernelError } from "./debt_kernel_error";
+import { RepaymentRouterError } from "./repayment_router_error";
+
+export interface ContractAddressBook {
```
Comment (nadavhollander) [inline src/types/error_parser.ts]: I like this nomenclature -- we should use it elsewhere.  No requested change here, but we can discuss this in the future.

File: src/types/error_parser.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    DebtKernel,
+    RepaymentRouter,
+}
+
+export class ErrorParser {
+    constructor(private contractAddressBook: ContractAddressBook) {
+        this.contractAddressBook = contractAddressBook;
+    }
+
+    public parseDecodedLogs = (logs: any[]): string[] => {
```
Comment (nadavhollander) [inline src/types/error_parser.ts]: I'd prefer if we kept the syntax we use to define function signatures consistent across the codebase, i.e. these signatures should look more like:

```javascript
public parseDecodedLogs(logs: any[]): string[] {
...
}
```

Comment (saturnial) [reply src/types/error_parser.ts]: I'm totally with you, except that this is where typescript can't solve the underlying crap that is javascript. I need to use arrow functions here to ensure that `this` is being scoped properly.

See: https://medium.freecodecamp.org/learn-es6-the-dope-way-part-ii-arrow-functions-and-the-this-keyword-381ac7a32881

File: src/types/error_parser.ts (hunk 1 above)
Comment (saturnial) [reply src/types/error_parser.ts]: I like it too. Thought it was clever ;) Glad to know I wasn't the only one.

File: src/types/error_parser.ts (hunk 2 above)
Comment (nadavhollander) [reply src/types/error_parser.ts]: Okay, fair enough :) . I'm sufficiently convinced this is cleaner than doing `.bind(this)` in the constructor.

Comment (saturnial) [reply src/types/error_parser.ts]: +1

**Pull request merged**